code/modules/research/xenoarchaeology/artifact/effects/unknown_effect_deadharvest.dm

```diff
@@ -0,0 +1,146 @@
+/datum/artifact_effect/deadharvest
+	effecttype = "deadharvest"
+	var/list/mob_spawn = list()
+	var/points = 0
+	var/can_be_controlled = 0
+	var/mob/living/controller //Whomever is the leader of these brainless minions
+
+/datum/artifact_effect/deadharvest/New()
+	..()
+	effect = pick(EFFECT_AURA, EFFECT_PULSE)
+	can_be_controlled = pick(0,1)
+	if(!mob_spawn.len)
+		new_mob_spawn_list()
+
+/datum/artifact_effect/deadharvest/proc/new_mob_spawn_list(var/choice)
+	if(!choice)
+		choice = rand(4)
+	mob_spawn = list()
+	switch(choice)
+		if(1) //Regular necromancy mobs
+			mob_spawn = list(/mob/living/simple_animal/hostile/necro/zombie = 100,
+						/mob/living/simple_animal/hostile/necro/skeleton = 100,)
+		if(2) //Infection zombies
+			mob_spawn = list(/mob/living/simple_animal/hostile/necro/zombie/turned = 50,
+						/mob/living/simple_animal/hostile/necro/zombie/rotting = 75,
+						/mob/living/simple_animal/hostile/necro/zombie/putrid = 100,
+						/mob/living/simple_animal/hostile/necro/zombie/crimson = 250,)
+		if(3) //Necromorphs
+			mob_spawn = list(/mob/living/simple_animal/hostile/monster/necromorph = 150,) //More necromorph mobs when?
+		if(4) //Randomized mobs
+			var/max_mobs = rand(3,7)
+			for(var/i=0,i<max_mobs,i++)
+				var/mob/living/simple_animal/mob_to_add = pick(typesof(/mob/living/simple_animal))
```

Comment:
It's the same deal except it cuts out potentially broken things without icon and such.